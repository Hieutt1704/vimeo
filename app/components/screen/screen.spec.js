import React from "react"
import { Screen } from "./screen"
import { render, screen } from "@testing-library/react-native"

test("renders fixed", () => {
  render(<Screen preset="fixed" />)
  expect(screen.toJSON()).toMatchSnapshot()
})

test("renders scroll", () => {
  render(<Screen preset="scroll" />)
  expect(screen.toJSON()).toMatchSnapshot()
})

test("renders auto", () => {
  render(<Screen preset="auto" />)
  expect(screen.toJSON()).toMatchSnapshot()
})
