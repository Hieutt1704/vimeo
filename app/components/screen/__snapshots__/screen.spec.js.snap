// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders auto 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "flex": 1,
          "height": "100%",
        },
        Object {},
      ],
      Object {
        "paddingBottom": 0,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "flex": 1,
          "height": "100%",
        },
        Object {},
        Object {
          "paddingTop": 0,
        },
      ]
    }
  >
    <RCTScrollView
      contentContainerStyle={
        Array [
          Object {
            "alignItems": "stretch",
            "justifyContent": "flex-start",
          },
          Object {},
        ]
      }
      keyboardShouldPersistTaps="handled"
      onContentSizeChange={[Function]}
      scrollEnabled={true}
      style={
        Array [
          Object {
            "backgroundColor": "#ffffff",
            "flex": 1,
            "height": "100%",
          },
          Object {},
        ]
      }
    >
      <View />
    </RCTScrollView>
  </View>
</View>
`;

exports[`renders fixed 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "flex": 1,
          "height": "100%",
        },
        Object {},
      ],
      Object {
        "paddingBottom": 0,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "stretch",
          "height": "100%",
          "justifyContent": "flex-start",
          "width": "100%",
        },
        Object {},
        Object {
          "paddingTop": 0,
        },
      ]
    }
  />
</View>
`;

exports[`renders scroll 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "flex": 1,
          "height": "100%",
        },
        Object {},
      ],
      Object {
        "paddingBottom": 0,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "flex": 1,
          "height": "100%",
        },
        Object {},
        Object {
          "paddingTop": 0,
        },
      ]
    }
  >
    <RCTScrollView
      contentContainerStyle={
        Array [
          Object {
            "alignItems": "stretch",
            "justifyContent": "flex-start",
          },
          Object {},
        ]
      }
      keyboardShouldPersistTaps="handled"
      scrollEnabled={true}
      style={
        Array [
          Object {
            "backgroundColor": "#ffffff",
            "flex": 1,
            "height": "100%",
          },
          Object {},
        ]
      }
    >
      <View />
    </RCTScrollView>
  </View>
</View>
`;
